posts/2024/containerize-visualizations-event.md

```diff
@@ -0,0 +1,42 @@
+---
+author: John Clyne
+date: 2024-05-07
+tags: tutorial hackathon events
+---
+
+# Workshop - From Jupyter Notebook to Web Server: Containerizing Interactive Visualizations 
+
+The [ESDS Events Working Group](https://ncar.github.io/esds/about/#events-working-group) is hosting a hybrid workshop on June 20th, from 9am until noon. 
+
+
+The event will be hosted at the Mesa Lab, located at 1850 Table
+Mesa Dr, Boulder, CO 80305. In order to attend the event virtually,
+registered participants will receive an email from Taysia
+Peterson roughly one week prior to the event with Zoom details.
+
+## Registration
+
+*Coming soon*
+
+## What is it?
+
+This workshop guides participants through the process of transforming
+an interactive visualization from a Jupyter Notebook into a
+Python-based web server. Attendees will learn how to create and run
+a containerized version of the web server, gaining practical skills
+in containerization and deployment. Additionally, the workshop will
+explore automation techniques for building container images and
+introduce hosting options available via the CISL On-premise Cloud
+Pilot, empowering participants to leverage cloud infrastructure for
+their projects. Here is a [link to the code
+repository](https://github.com/NicholasCote/nbviz-to-container)
+containing a notebook that walks through the workshop content.
+
+## Who should attend?
```

Comment:
Is this open to folks external to NSF NCAR & UCAR then?

Reply:
Good catch. No, it is not. Only UCAR staff. I've updated the text